### DISCUSSION TITLE
DNS Direct IP

### DISCUSSION THREAD

**Discussion by lublak** [rustdesk/rustdesk#2454] [Q&A]
I have a pc with the name: DESKTOP-BSCASF.
Now i try to use this name to connect to the pc. But it doesn't work.
Ip works. Can i enable dns resolution somehow?

**Comment by pedjas** [comment]
Try DESKTOP-BSCASF.local
That should use mDNS and provide IP.

**Reply by lublak** [reply]
@pedjas thanks for the answer. I tried it on my ipad but it shows me: ID not exists.
